esp/scm/ws_fs.ecm

```diff
@@ -93,7 +93,7 @@ ESPStruct [nil_remove] DFUWorkunit
     [min_ver("1.09")] bool recordStructurePresent(false);
     [min_ver("1.10")] bool quotedTerminator(true);
     [min_ver("1.12")] bool preserveCompression(true);
-    [min_ver("1.14")] int expireDays;
+    [min_ver("1.14")] int expireDays(-1);
```

Comment:
not sure this is right. Should be left unset I think.
The esp code should check if set and only use it if it is.
It looks like some of the code does, e.g. in CFileSprayEx::onCopy
```
        if (!req.getExpireDays_isNull())
            wuOptions->setExpireDays(req.getExpireDays());
```

but other code doesn't, e.g. in CFileSprayEx::onSprayFixed

Reply:
getExpireDays_isNull() is added to onSprayXXXX() methods.

Reply:
-1 removed